src/ceph-volume/ceph_volume/tests/functional/lvm/playbooks/test_bluestore.yml

```diff
@@ -67,6 +67,16 @@
       environment:
         CEPH_VOLUME_DEBUG: 1
 
+    - name: node inventory
```

Comment:
is this double call to the inventory sub-command a copy/paste error or do you really need to run it twice

Reply:
c/p error indeed...my apologies.